pkg/chartutil/create.go

```diff
@@ -74,13 +74,13 @@ ingress:
     # - secretName: chart-example-tls
     #   hosts:
     #     - chart-example.local
-resources:
-  limits:
-    cpu: 100m
-    memory: 128Mi
-  requests:
-    cpu: 100m
-    memory: 128Mi
+resources: {}
+  # limits:
```

Comment:
If we are commenting out this block, we should leave a comment about how the user can uncomment this block if they want to specify resources